packages/material-renderers/src/util/theme.ts

```diff
@@ -33,3 +40,29 @@ export interface JsonFormsTheme extends Theme {
     };
   };
 }
+
+export type WithInputProps = {
+  InputComponent: React.JSXElementConstructor<any>;
+  label?: string;
+};
+
+const variantToInput = {
+  standard: Input,
+  filled: FilledInput,
+  outlined: OutlinedInput,
+};
+
+export function useInputComponent(props: unknown): {
```

Comment:
Why is this typed as `unknown` when this is later just assumed to be of type `TextFieldProps` in line 60 without any additinal checks. Shouldn't we just required `TextFieldProps` as the type for props?

Reply:
I am not sure. It was late when the PR  was crafted. The need of passing props to the hook can probably be avoided. I will try to simplify it.